CP01CNSM/CP01CNSM_R00009_ingest.csv

```diff
@@ -1,31 +1,31 @@
 mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl37/CTDBP*/SBE*.hex,CP01CNSM-MFD37-03-CTDBPD000,recovered_inst,Expected,
 mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl37/dosta2/*dosta2.log,CP01CNSM-MFD37-04-DOSTAD000,recovered_host,Expected,
 mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl37/zplsc/*.zplsc.log,CP01CNSM-MFD37-07-ZPLSCC000,recovered_host,Expected,
-mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl37/ZPLSC*/*/*.01A,CP01CNSM-MFD37-07-ZPLSCC000,recovered_inst,Expected,
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl37/ZPLSC*/*/*/*.01A,CP01CNSM-MFD37-07-ZPLSCC000,recovered_inst,Expected,Test parser
 mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/cpm2/syslog/cpm_status.txt,CP01CNSM-RIC21-00-CPMENG000,recovered_host,Expected,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl26/syslog/*.syslog.log,CP01CNSM-RID26-00-DCLENG000,recovered_host,Expected,
 mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl26/velpt1/*.velpt1.log,CP01CNSM-RID26-04-VELPTA000,recovered_host,Expected,
 mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl26/VELPT_snAQD-*/*.aqd,CP01CNSM-RID26-04-VELPTA000,recovered_inst,Expected,
 mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl26/phsen1/*.phsen1.log,CP01CNSM-RID26-06-PHSEND000,recovered_host,Expected,
-mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl26/PHSEN*/SAMI_P0095_*.txt ,CP01CNSM-RID26-06-PHSEND000,recovered_inst,Expected,
-mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl26/nutnr/*.nutnr.log,CP01CNSM-RID26-07-NUTNRB000,recovered_host,Expected,
-mi.dataset.driver.nutnr_b.nutnr_b_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl26/NUTNR_sn*/SCH*.DAT,CP01CNSM-RID26-07-NUTNRB000,recovered_inst,Expected,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl26/PHSEN*/SAMI_*.txt,CP01CNSM-RID26-06-PHSEND000,recovered_inst,Expected,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl26/nutnr/*.nutnr.log,CP01CNSM-RID26-07-NUTNRB000,recovered_host,Expected,
+mi.dataset.driver.suna.suna_instrument_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl26/NUTNR_sn*/DAT/*.CSV,CP01CNSM-RID26-07-NUTNRB000,recovered_inst,Expected,

```

Comment:
This deployment had a SUNA. Should use SUNA parsers. Fixed filemasks to point to SUNA data